src/databases/IDX/CMakeLists.txt

```diff
@@ -57,7 +56,19 @@ ${VISIT_INCLUDE_DIR}/visit_vtk/lightweight
 ${VTK_INCLUDE_DIRS}
 )
 
-LINK_DIRECTORIES(${VISIT_LIBRARY_DIR} )
+### Note: This reader always requires MPI
+
+# Use same MPI libs as Visit
+IF(VISIT_PARALLEL)
```

Comment:
Since in src/databases/CMakeLists.txt, this reader is only included if VISIT_PARALLEL is satisfied, I don't think this extra 'if VISIT_PARALLEL' logic is necessary. It doesn't hurt anything, though.